docs-authoring-pack/package.json

```diff
@@ -23,6 +23,7 @@
   ],
   "extensionDependencies": [
     "DavidAnson.vscode-markdownlint",
+    "docsmsft.docs-images",
```

Comment:
@IEvangelist - Can we just take this out for the moment? We have a CI/CD pipeline process we need to setup for this new extension. Once we get that setup, we will put this into the Docs Authoring Pack, and publish it to the marketplace. Sound good?

Reply:
Yes, absolutely. I'll update it today.